apps/onlyoffice/8.2.1.1/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   onlyoffice:
-    image: onlyoffice/documentserver:8.2.1.1
+    image: onlyoffice/documentserver:8.2.2.1
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
There are no major changes detected in the provided code snippet between version 8.2.1.1 and 8.2.2.1. However, it's worth noting that there may be small variations due to minor adjustments made since versions were released.

In order to ensure consistency with respect to Docker specifications and images, you might want to:

- Define all `networks` options using their short form (e.g., `networks: []`) instead of full description to keep things concise.
- Ensure your service name is consistent across different environments like `CONTAINER_NAME`, etc.
- If not required, remove unused labels such as `restart: always`.

As per my current knowledge cutoff date of Sept 1st, 2021, this change should generally work fine without breaking compatibility on newer machines but could possibly introduce differences when compared against earlier releases based on any subtle internal optimizations.